04_numeronym/hsy3418/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+//numeronyms implements a function that returns a slice of numeronyms for its input strings
```

Comment:
This comment is a little verbose. The "implements a function" part does not really add anything meaningful as we can see it is a function. Just describe what it does:
```
// numeronyms returns a slice of numeronyms for its input strings.
```
Also note, the full-stop at the end. Write these as complete sentences as they can be extracted as documentation.